gradle/libs.versions.toml

```diff
@@ -203,6 +203,6 @@ dependencygraph = { id = "com.savvasdalkitsis.module-dependency-graph", version.
 dependencycheck = { id = "org.owasp.dependencycheck", version.ref = "dependencycheck" }
 dependencyanalysis = { id = "com.autonomousapps.dependency-analysis", version.ref = "dependencyanalysis" }
 paparazzi = "app.cash.paparazzi:1.3.1"
-sonarqube = "org.sonarqube:4.3.1.3277"
+sonarqube = "org.sonarqube:4.2.1.3168"
```

Comment:
Maybe add a comment above that `4.3.1.3277` is problematic, because Renovate will try to upgrade again.